### DISCUSSION TITLE
Welcome to noah-mqtt Discussions!

### DISCUSSION THREAD

**Discussion by mtrossbach** [mtrossbach/noah-mqtt#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by robinjoms86** [comment]
First of all, thank you very much for your contribution. Before yours, there was no project extracting this data in a co,fortable manner.
Due to the possibility to set the default output power of the noah it is possible to realise no consumption at all, dynamically.

Is there a way to get the differenciated parameters from pv modules connected to the noah, i.e. Watt PV1 and PV2?

One more feature could be to set the timetable & LF/BF parameters.

Another idea is to implement set parameters for the NEO inverter connected to the NOAH

**Reply by mtrossbach** [reply]
Thank you so much for your kind words! I'm glad to hear that you're finding the project useful.

Regarding the differentiated parameters for PV1 and PV2, to my knowledge, it isn't possible to retrieve those values as the device does not collect them. However, if you've seen these values available in the app or anywhere else, please let me know. If they are accessible, I might be able to find a way to fetch them.

As for the timetable and LF/BF parameters, they're already on my to-do list, so stay tuned!

Unfortunately, since I don't have a NEO inverter, I'm currently unable to experiment with controlling it. But I appreciate the suggestion—it's definitely something I'd consider adding in the future.

Thanks again for your feedback! If you have any other ideas or questions, feel free to share.

**Comment by cgatnet** [comment]
Hi there, 
I just wanted to say: "**Thank you very much!**", because I tried your Docker today with my NOAH 2000 (connected to a Hoymiles inverter) and it worked without issues. This is so helpful to now have the SOC in my Smart Home informations. :o)
I had a short "wtf?", because it failed with "panic, not authenticated", but this was just because my mqtt server needed the user and password as addition in the command line of Docker startup.  👍 
Hope "noah-mqtt" will work at least as long as the project "grott" needs to decode the NOAH 2000 communication protocol (if ever), because I would love to not need to go online to get the information that is already in my network.
